association_backend/association/api/query/query.py

```diff
@@ -0,0 +1,34 @@
+import strawberry
+
+from association.api.context import Info
+from association.domain.entities import SubscriptionState
+from association.settings import TEST_USER_ID
+
+
+@strawberry.type
+class HasAssociationSubscriptionResponse:
+    has_association_subscription: bool
+
+
+@strawberry.type
+class Query:
+    @strawberry.field()  # permission_classes=[IsJWTAvailable])
+    async def has_association_subscription(
```

Comment:
note: this is not correct, you have to extend the user type not expose a new query like this

I will try to find an example on how to extend types, but probably it's best to write this down somewhere as a TODO and remove this code for now. We can implement it in another PR I think :)

Reply:
I know, my idea was to delete it after implementing the type extension. I'm deleting it.
I moved this logic into a service.